### PR TITLE
Remove lodash entirely (it was only used for two functions). Related to gh-490

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

**Comment by rwaldron** [comment]
The travis failure is completely bogus. This was the same issue that @Frijol @LinusU's patches were hitting.

**Comment by rwaldron** [comment]
I reran the tests, all are passing.

**Comment by HipsterBrown** [comment]
Rad! :+1:

File: test/unit/controller.js [hunk 1]
```diff
@@ -578,7 +578,7 @@ exports['Tessel.get'] = {
         test.equal(this.closeTesselConnections.callCount, 1);
         test.equal(Array.isArray(this.closeTesselConnections.args[0]), true);
         test.equal(this.logsInfo.callCount, 2);
-        test.equal(_.contains(this.logsInfo.lastCall.args[0], 'the_name'), true);
+        test.equal(this.logsInfo.lastCall.args[0].includes('the_name'), true);
```
Comment (LinusU) [inline test/unit/controller.js]:
```text
➜  test  node --version
v5.5.0
➜  test  node 
> [1, 2, 3].includes(2)
TypeError: [1,2,3].includes is not a function
    at repl:1:11
    at REPLServer.defaultEval (repl.js:252:27)
    at bound (domain.js:287:14)
    at REPLServer.runBound [as eval] (domain.js:300:12)
    at REPLServer.<anonymous> (repl.js:417:12)
    at emitOne (events.js:95:20)
    at REPLServer.emit (events.js:182:7)
    at REPLServer.Interface._onLine (readline.js:211:10)
    at REPLServer.Interface._line (readline.js:550:8)
    at REPLServer.Interface._ttyWrite (readline.js:827:14)
```

Am I missing something, I didn't think that `includes` was landed in Node.js yet, I think it's only stage 4.

[`array-includes`](https://www.npmjs.com/package/array-includes) is a nice replacement :+1:

Comment (rwaldron) [inline test/unit/controller.js]: You are correct, however the value being tested here is a string ;)

Comment (LinusU) [inline test/unit/controller.js]: Ahahaha :)

Awesome though, `String.prototype.includes` is supported in 4.0 as well :raised_hands:

**Comment by LinusU** [comment]
LGMT :raised_hands:

**Pull request merged**